git-reqs

```diff
@@ -18,29 +18,53 @@ def init(args):
 
 
 def import_reqs(args):
-    module_path = args.project_root + '/' + args.module
     project = requirement_module(args.project_root)
-    req_module = project.modules[args.module]
+    if args.module:
+        req_module = project.modules[args.module]
+        module_path = args.project_root + '/' + args.module
+    else:
+        req_module = project
+        module_path = args.project_root
+
+    if args.file:
```

Comment:
These lines are used in multiple places. I think a seperate function should be created and called